.travis.yml

```diff
@@ -15,7 +16,8 @@ env:
   - CHEF_VERSION=12.4.1
   - CHEF_VERSION=12.3.0
   - CHEF_VERSION=12.2.1
-  - CHEF_VERSION=12.1.2
+  # - CHEF_VERSION=12.2.0
+  # - CHEF_VERSION=12.1.2    # known broken for now.
```

Comment:
I'll clean that up: I'm thinking we don't need 12.2.0 if we've got 12.2.1, and 12.1.2 is broken but we can run it anyway. I'll also re-add Ruby versions to test against.